boot/bootdata/livecd/unattend.inf

```diff
@@ -11,3 +11,7 @@ UnattendSetupEnabled = no
 ; see hivesys.inf for available languages
 LocaleID = 409
 
+; set product option
+; 0: ReactOS Server
+; 1: ReactOS Workstation
+ProductOption = 0
```

Comment:
At the moment livecd does not have handling of product option at all, so it is not needed in the unattend.inf for the livecd either :)

Reply:
Reverted.